tools/cldr-code/src/main/java/org/unicode/cldr/util/Annotations.java

```diff
@@ -679,25 +854,54 @@ private static void writeList() {
         final UnicodeMap<Annotations> map = eng.getUnresolvedExplicitValues();
         Set<String> keys = new TreeSet<>(ChartAnnotations.RBC);
         map.keySet().addAllTo(keys);
-//        keys.add("👩🏻‍⚖");
+        //        keys.add("👩🏻‍⚖");
         for (String key : keys) {
-            System.out.println(Utility.hex(key, 4, "_").toLowerCase(Locale.ROOT)
-                + "\t" + key
-                + "\t" + map.get(key).getShortName()
-                + "\t" + Joiner.on(" | ").join(map.get(key).getKeywords()));
+            System.out.println(
+                Utility.hex(key, 4, "_").toLowerCase(Locale.ROOT) +
+                "\t" +
+                key +
+                "\t" +
+                map.get(key).getShortName() +
+                "\t" +
+                Joiner.on(" | ").join(map.get(key).getKeywords())
+            );
         }
         for (String s : Arrays.asList(
-            "💏", "👩‍❤️‍💋‍👩",
-            "💑", "👩‍❤️‍👩",
-            "👪", "👩‍👩‍👧",
-            "👦🏻", "👩🏿",
-            "👨‍⚖", "👨🏿‍⚖", "👩‍⚖", "👩🏼‍⚖",
-            "👮", "👮‍♂️", "👮🏼‍♂️", "👮‍♀️", "👮🏿‍♀️",
-            "🚴", "🚴🏿", "🚴‍♂️", "🚴🏿‍♂️", "🚴‍♀️", "🚴🏿‍♀️")) {
+            "💏",
+            "👩‍❤️‍💋‍👩",
+            "💑",
+            "👩‍❤️‍👩",
+            "👪",
+            "👩‍👩‍👧",
+            "👦🏻",
+            "👩🏿",
+            "👨‍⚖",
+            "👨🏿‍⚖",
+            "👩‍⚖",
+            "👩🏼‍⚖",
+            "👮",
+            "👮‍♂️",
+            "👮🏼‍♂️",
+            "👮‍♀️",
+            "👮🏿‍♀️",
+            "🚴",
+            "🚴🏿",
+            "🚴‍♂️",
+            "🚴🏿‍♂️",
+            "🚴‍♀️",
+            "🚴🏿‍♀️"
+        )) {
             final String shortName = eng.getShortName(s);
             final Set<String> keywords = eng.getKeywords(s);
-            System.out.println("{\"" + s + "\",\"" + shortName + "\",\"" + Joiner.on("|")
-                .join(keywords) + "\"},");
+            System.out.println(
+                "{\"" +
+                s +
+                "\",\"" +
+                shortName +
+                "\",\"" +
+                Joiner.on("|").join(keywords) +
+                "\"},"
+            );
```

Comment:
very much expanded

Reply:
right -- compare this version of the same code in my other PR https://github.com/unicode-org/cldr/pull/1913:
```
            System.out.println(
                "{\"" + s + "\",\"" + shortName + "\",\"" + Joiner.on("|").join(keywords) + "\"},"
            );
```
if you like that better, that's an argument for printWidth 100

Reply:
i was just looking for a width parameter! Yes, let's do printwidth 100

Reply:
the width is the only real issue i have otherwise lgtm

Reply:
great, lets see what @macchiati and others think